app/api/stripe/webhook/route.ts

```diff
@@ -71,6 +78,34 @@ export async function POST(request: Request) {
 
 async function handleVargasJrHired(event: Stripe.Event) {
   console.log("Handling Vargas Jr hired event:", event.id);
+  
+  try {
+    const session = event.data.object as Stripe.Checkout.Session;
+    console.log("Processing checkout session:", session.id);
+    
+    console.log("Generating hiring agreement PDF...");
+    const pdfBuffer = await generateHiringAgreementPDF({
+      employeeName: session.customer_details?.name || undefined,
+      position: "Senior Software Developer",
+      startDate: new Date().toLocaleDateString(),
+      salary: "$150,000 USD",
```

Comment:
We should get this value from the subscription's price found on the Checkout session